### ISSUE TITLE
CLI - Adding proxy support 

### ISSUE THREAD

**Issue by paulolai** [rangav/thunder-client-support#1089]
**Please describe feature/problem details and solution you'd like.**
I can't find a way to configure proxy settings, the extension just works. It doesn't seem to use the HTTP_PROXY and HTTPS_PROXY environmental variables, there's no details of how to set them.

**Describe alternatives you've considered**

**Your Team Size Using TC:**
0 - can't use it yet without proxy support

**Comment by rangav** [comment]
Hi @paulolai 

You need to use Vscode proxy setting
https://github.com/rangav/thunder-client-support#proxy

Make sure the setting is `workspace scope`

**Comment by paulolai** [comment]
I should've been more explicit sorry. I have no trouble using the VSCode extension, it picked up the system's settings without trouble. The CLI however doesn't pick up those settings. I just get the following even after setting the VSCode proxy setting:

```
Request URL:  GET - https://www.thunderclient.com/welcome

Status: ERROR    Size: 0 B    Time: 0 ms

Response
Couldn't resolve the hostname to an IP address, Verify Url: 
https://www.thunderclient.com/welcome
```

**Comment by rangav** [comment]
This feature is now implemented and published to marketplace, please update to `v2.6.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.6.0

Please let me know your feedback.
